css/my_website.css

```diff
@@ -0,0 +1,52 @@
+body{
+    margin:0px;
+    background-color:rgb(231, 231, 231);
+    color:rgb(65, 65, 65);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size:20px;
+    line-height:normal;
+}
+.main-header{
+    background-color:#003F87;
+    color:#ffffff;
+    height: 75px;
+    padding: 0em;
+    width:100%;
+    overflow: hidden;
+}
+.text{
```

Comment:
Figure out what attributes should be applied to all these paragraphs and select the element type instead of having to add a class to every single paragraph

Reply:
I did the one paragraph class is that what you meant?

Reply:
Yes, I like the new CSS